server/app/services/ambassadors.js

```diff
@@ -154,9 +157,94 @@ async function signup(json, verification, carrierLookup) {
   if (existing_tripler) {
     new_ambassador.relateTo(existing_tripler, "was_once")
   }
+
+  setAmbassadorHubspotID(new_ambassador)
+
   return new_ambassador
 }
 
+/*
+ * setAmbassadorHubspotID(ambassador)
+ */
+async function setAmbassadorHubspotID(ambassador) {
+  //only continue if there's no hs_id
+  console.log("ambassador id:", ambassador.get("id"))
+  console.log("ambassador hs_id:", ambassador.get("hs_id"))
+
+  if (!ambassador.get("hs_id")) {
+    console.log("no hs id, gettig it from hs")
+    const hs_response = await getAmbassadorHSID(ambassador.get("email"))
+    let cypher_response = await neode.cypher(
+      "MATCH (a:Ambassador {id: $id}) SET a.hs_id=toInteger($hs_id) RETURN a.first_name, a.hs_id",
+      {
+        id: ambassador.get("id"),
+        hs_id: hs_response,
+      },
+    )
+
+    // send initial update to hubspot
+
+    let obj = {}
+    ;[
+      "first_name",
+      "last_name",
+      "approved",
+      "quiz_completed",
+      "onboarding_completed",
+      "alloy_person_id",
+    ].forEach((x) => (obj[x] = ambassador.get(x)))
+    obj["hs_id"] = hs_response
+    updateHubspotAmbassador(obj)
+  }
+  return ambassador.get("hs_id")
+}
+
+/*
+ * updateAmbassadorTriplerInfoHubspot(ambassador)
+ */
```

Comment:
This could also use a better name or a comment, e.g. 

```
sendTriplerCountsToHubspot(ambassador)

// If the ambassador node has an `hs_id`, sends the count of the number
// of triplers with each status over to the HubSpot record for this
// ambassador.
```